public/app/dashboard/profile.html

```diff
@@ -148,29 +148,26 @@ <h4 class="profile-activity-breakdown-header">Community Impacts</h4>
   </div>
 
   <div class="profile-assets-container">
-    <div class="profile-swimlane-filters">
+    <div class="profile-filters">
       <h3>{{isMyProfile ? 'My Assets' : 'Assets'}}{{user.totalAssetsInCourse ? ':' : ''}}</h3>
 
-      <div data-ng-repeat="(filter, label) in user.assets.filterLabels" data-ng-if="user.totalAssetsInCourse">
+      <div class="profile-filter-label" data-ng-repeat="(filter, label) in user.assets.filterLabels" data-ng-if="user.totalAssetsInCourse">
         <button type="button"
           id="user-assets-by-{{filter}}"
-          class="btn btn-link profile-swimlane-filters-label"
+          class="btn btn-link profile-filter-label"
           data-ng-click="sortUserAssets(filter)"
           data-ng-disabled="user.assets.sortBy === filter"
           data-ng-bind="label"></button>
+        <span data-ng-if="user.totalAssetsInCourse && (user.assets.sortBy === filter) && (user.assets.total > maxPerSwimlane)">({{maxPerSwimlane}} of
```

Comment:
@johncrossman Can the `user.totalAssetsInCourse` conditional be removed here since it exists in the wrapping div?